app/assets/javascripts/actions/tickets_actions.js

```diff
@@ -130,4 +132,19 @@ export const deleteTicket = id => async (dispatch) => {
   dispatch({ type: DELETE_TICKET, id });
 };
 
-export const setTicketFilters = filters => ({ type: FILTER_TICKETS, filters });
+export const setTicketOwnersFilter = filters => ({ type: FILTER_TICKETS, filters: { owners: filters } });
+export const setTicketStatusesFilter = filters => ({ type: FILTER_TICKETS, filters: { statuses: filters } });
+
+export const setInitialTicketFilters = () => (dispatch, getState) => {
+  // Open tickets only
+  dispatch(setTicketStatusesFilter([{ value: TICKET_STATUS_OPEN, label: STATUSES[TICKET_STATUS_OPEN] }]));
+
+  // Owned by current user, or no one
+  const state = getState();
+  const currentUserId = state.currentUserFromHtml.id;
+  const currentUser = state.admins.find(admin => admin[1] === currentUserId);
```

Comment:
This is a little confusing on first read. It might be better as:
```js
const currentUser = state.admins.find(([ _username, id ]) => id === currentUserId);
```

You could go even further and apply destructuring to remove the next line:
```js
const [ label, value ] = state.admins.find(([ _username, id ]) => id === currentUserId);
```

Reply:
Looks good. Done.